ocf_datapipes/transform/xarray/pv_power_rolling_window.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional, Union
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+from torchdata.datapipes import functional_datapipe
+from torchdata.datapipes.iter import IterDataPipe
+
+
+@functional_datapipe("pv_power_rolling_window")
+class PVPowerRollingWindowIterDataPipe(IterDataPipe):
+    def __init__(
+        self,
+        source_dp: IterDataPipe,
+        window: Union[int, pd.tseries.offsets.DateOffset, pd.core.indexers.objects.BaseIndexer] = 3,
+        min_periods: Optional[int] = 2,
+        center: bool = True,
+        win_type: Optional[str] = None,
```

Comment:
win=window?

Reply:
I'm not sure, I just copied this in from PP, I think so, @JackKelly would know more.